From 81fb045eca1e3bb8760f8b8c437d01a3e56c8ce9 Mon Sep 17 00:00:00 2001
From: Michael Stahl <mstahl@redhat.com>
Date: Mon, 17 Nov 2014 15:46:23 +0100
Subject: [PATCH] officecfg: set VCL.WM.ShouldSwitchWorkspace to false

It's quite obnoxious of applications to switch virtual desktops, and
i#45160 / i#96684 indicate the main reason for this was to avoid crashes
9 years ago; let's see if we get any crash reports with it disabled.

Change-Id: Ib020380c9ad3b386c7cf00fb2144cb766d1f6c83
(cherry picked from commit d93c89ddf564d0445f28e0ba552ba94975d40f2a)
---
 officecfg/registry/data/org/openoffice/VCL.xcu | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/officecfg/registry/data/org/openoffice/VCL.xcu b/officecfg/registry/data/org/openoffice/VCL.xcu
index ec48498..e5b56e8 100644
--- a/officecfg/registry/data/org/openoffice/VCL.xcu
+++ b/officecfg/registry/data/org/openoffice/VCL.xcu
@@ -51,7 +51,7 @@
     </node>
     <node oor:name="WM" oor:op="replace">
       <prop oor:name="ShouldSwitchWorkspace" oor:op="replace" oor:type="xs:string">
-        <value/>
+        <value>false</value>
       </prop>
     </node>
   </node>
-- 
1.9.3

